src/components/banner/banner.config.yml

```diff
@@ -36,6 +38,8 @@ variants:
           Un <strong>candado</strong> ($LOCK_IMG) o <strong>https://</strong> significa que usted se conectó de forma segura a un sitio web $TLD.  Comparta información sensible sólo en sitios web oficiales y seguros.
   - name: dot-mil
     context:
+      banner:
+        aria_label: "Official government website,"
```

Comment:
... and use trailing commas as a hack to provide legible ARIA labels but still make them unique